### ISSUE TITLE
Error when fetching sesssions

### ISSUE THREAD

**Issue by dfalling** [ZennerIoT/pow_postgres_store#1]
Hi, I think I've followed all the steps of the docs, but am having an issue when Pow attempts to fetch sessions:
```
** (exit) an exception was raised:
    ** (CaseClauseError) no case clause matching: 20
        (pow_postgres_store) lib/pow_postgres_store.ex:141: anonymous fn/2 in Pow.Postgres.Store.filter_key_match/2
        (elixir) lib/enum.ex:1948: Enum."-reduce/3-lists^foldl/2-0-"/3
        (pow_postgres_store) lib/pow_postgres_store.ex:98: Pow.Postgres.Store.all/2
        (pow) lib/pow/store/credentials_cache.ex:46: Pow.Store.CredentialsCache.fetch_sessions/3
        (pow) lib/pow/store/credentials_cache.ex:167: Pow.Store.CredentialsCache.do_delete_user_sessions_with_fingerprint/3
        (pow) lib/pow/store/credentials_cache.ex:78: Pow.Store.CredentialsCache.put/3
        (pow) lib/pow/plug/session.ex:192: anonymous fn/6 in Pow.Plug.Session.before_send_create/3
        (elixir) lib/enum.ex:1948: Enum."-reduce/3-lists^foldl/2-0-"/3
        (plug) lib/plug/conn.ex:1563: Plug.Conn.run_before_send/2
        (plug) lib/plug/conn.ex:404: Plug.Conn.send_resp/1
```

Is there I step I missed that would cause this match error?

Thanks!

**Comment by gdub01** [comment]
I had the same error when I attempted to login a user.

**Comment by narrowtux** [comment]
Sorry for the late reply, 

I kinda forgot that normally integers are used for IDs (we always use UUIDs which are strings in Elixir).

The fix shouldn't take too much effort, only add a `or is_integer(val)` to the guard clause

**Comment by narrowtux** [comment]
Should be fixed in `1.0.0-rc2`

**Comment by dfalling** [comment]
It works great, thanks!
